integration-cli/docker_cli_push_test.go

```diff
@@ -64,9 +59,7 @@ func (s *DockerRegistrySuite) TestPushMultipleTags(c *check.C) {
 
 	// Ensure layer list is equivalent for repoTag1 and repoTag2
 	out1, _ := dockerCmd(c, "pull", repoTag1)
-	if strings.Contains(out1, "Tag t1 not found") {
-		c.Fatalf("Unable to pull pushed image: %s", out1)
-	}
+
```

Comment:
https://github.com/docker/docker/pull/16971#discussion_r41832566

Not sure about this one : Is there a way the message "Unable to pull pushed image" would be in output without an error ?